src/components/InputTime/InputTime.module.css

```diff
@@ -0,0 +1,29 @@
+.InputTime {
+  position: relative;
+  display: inline-flex;
+  flex-flow: row nowrap;
+  padding: 0;
+}
+
+.Input.Input {
```

Comment:
This suppresses the nested `Input`'s non-layout-related styles so that the `InputTime` styles are the only border / background / etc.